components/server/src/prometheusMetrics.ts

```diff
@@ -0,0 +1,19 @@
+import * as prometheusClient from 'prom-client';
+
+// Enable collection of default metrics.
+prometheusClient.collectDefaultMetrics({ timeout: 5000 });
+export const register = prometheusClient.register
```

Comment:
All metrics coming from `server` will have the same endpoint and use the same register, i.e., they'll all be exposed by the same endpoint `/metrics`.
We choose the register where a metric will be registered while describing the metric:
https://github.com/gitpod-io/gitpod/blob/4ffdf34f91da3333a19cc61fab404236bcea348d/components/server/src/prometheusMetrics.ts#L7-L12

This line here will be equal for all metrics we may add in the future: https://github.com/gitpod-io/gitpod/blob/4ffdf34f91da3333a19cc61fab404236bcea348d/components/server/src/prometheusMetrics.ts#L11

---
I exported the register just so it can be used here:
https://github.com/gitpod-io/gitpod/blob/4ffdf34f91da3333a19cc61fab404236bcea348d/components/server/ee/src/monitoring-endpoint-ee.ts#L20-L29